android-configure

```diff
@@ -16,5 +16,5 @@ export PLATFORM=android
 
 if [ $2 -a $2 == 'gyp' ]
   then
-    ./gyp_uv.py -Dtarget_arch=arm -DOS=android
+    ./gyp_uv.py -Dtarget_arch=arm -DOS=android -f make-android
```

Comment:
Yep it needs it. This sets the gyp build flavor to android. Without it cross compilation doesn't work as it tries to use some host settings. For some more insight into where it's used see ./build/gyp/pylib/gyp/generator/make.py and gyp.common.GetFlavor(params)